metricingester/obs.go

```diff
@@ -0,0 +1,18 @@
+package metricingester
+
+import (
+	"context"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/sirupsen/logrus"
+	"github.com/stripe/veneur/trace"
+)
+
+func traceLogger(log *logrus.Logger, ctx context.Context) *logrus.Entry {
+	if span, ok := opentracing.SpanFromContext(ctx).(*trace.Span); ok {
+		return log.
+			WithField("trace_id", span.TraceID).
+			WithField("span_id", span.SpanID)
```

Comment:
This looks interesting, what is it for / how does it look when logging? If it doesn't print hex numbers yet, you might want to make it do that - that's how LS presents them at least, so should make it easier to cross-link between tools.